nolearn/lasagne/handlers.py

```diff
@@ -24,7 +24,7 @@ def table(self, nn, train_history):
         info_tabulate = OrderedDict([
             ('epoch', info['epoch']),
             ('train loss', "{}{:.5f}{}".format(
-                ansi.BLUE if info['train_loss_best'] else "",
+                ansi.CYAN if info['train_loss_best'] else "",
```

Comment:
I reset the commit and rebased with current master. I don't know why it's still there :(